addons/web/static/src/js/core/service_mixins.js

```diff
@@ -238,43 +238,17 @@ var ServicesMixin = {
      * Displays a notification.
      *
      * @param {Object} options
-     * @param {string} options.title
+     * @param {string} [options.title]
      * @param {string} [options.subtitle]
      * @param {string} [options.message]
      * @param {string} [options.type='warning'] 'info', 'success', 'warning', 'danger' or ''
      * @param {boolean} [options.sticky=false]
      * @param {string} [options.className]
+     * @param {boolean} [options.isMessageHtml=false]
```

Comment:
maybe an imperative `messageIsHtml`? The `isX` form is more of a query e.g. checking for a widget's state or somesuch.

Reply:
Well for me the boolean is the answer to a question (Is message html?) which usually comes with the verb at the start.
Also helps to detect the type at first sight. Quickly reading something that starts with "message..." I would first think it contains a message (or something else, but I'd need to read everything more carefully).
Anyway as per my first comment, I might remove the possibility to use html all together if all it is ever used for is to pass a list.

Reply:
> Well for me the boolean is the answer to a question (Is message html?) which usually comes with the verb at the start.

Yes, but here it's not a question, the caller is telling the function that the message is HTML.

Reply:
Ok I don't mind either way, I renamed it to your suggestion for now.